packages/graph/hash_graph/lib/graph/src/api/web/error.rs

```diff
@@ -0,0 +1,37 @@
+#![allow(
+    clippy::needless_pass_by_value,
+    reason = "This is considered the last-stop for errors from the perspective of the web-api and \
+              taking Owned reports allows us to simply `map_err` and return the appropriate error \
+              response from request handlers"
+)]
```

Comment:
Do I see correctly, that *currently* it's not needed but it's easier to apply error handling in the future?